.github/ISSUE_TEMPLATE/reduce-code-complexity.md

```
@@ -0,0 +1,59 @@
+      placeholder: "Example: The `create_user` function has multiple conditionals, increasing cyclomatic complexity."
+    validations:
+      required: true
+
+  - type: textarea
+    id: suggested_fixes
+    attributes:
+      label: Suggested Fixes
+      description: "Provide possible ways to refactor and simplify the code."
+      placeholder: "Example: Extract the logic into separate functions, reduce redundant conditions, use Django’s built-in methods."
+
+  - type: textarea
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)